README.md

```diff
@@ -17,21 +17,28 @@ Given a probabilistic model, RxInfer allows for an efficient message-passing bas
 
 Turing comparison             |  Scalability performance
 :-------------------------:|:-------------------------:
 ![](benchmarks/plots/lgssm_comparison.svg?raw=true&sanitize=true)  |  ![](benchmarks/plots/lgssm_scaling.svg?raw=true&sanitize=true)
 
 ### Faster inference with better results
 
-RxInfer.jl not only beats generic-purpose Bayesian inference methods, executes faster, and scales better, but also provides more accurate results for various complex problems. Check out our [examples](https://biaslab.github.io/RxInfer.jl/stable/examples/overview/)!
+RxInfer.jl not only beats generic-purpose Bayesian inference methods in conjugate models, executes faster, and scales better, but also provides more accurate results. Check out the [documentation](https://biaslab.github.io/RxInfer.jl/stable/examples/overview/) for more examples!.
 
 Inference with RxInfer             |  Inference with HMC
 :-------------------------:|:-------------------------:
 ![](benchmarks/plots/inference_rxinfer.svg?raw=true&sanitize=true)  |  ![](benchmarks/plots/inference_turing.svg?raw=true&sanitize=true)
 
-The benchmark and accuracy experiment, which generated these plots, is available in the `benchmarks/` folder.
+The benchmark and accuracy experiment, which generated these plots, is available in the `benchmarks/` folder. Note, that the execution speed and accuracy 
+of the HMC heavily depends on the choice of hyper-parameters. `RxInfer`, in this example, performs consistent and exact inference and does not depent on any hyper-parameters.
+
+### References
+
+- [Variational Message Passing and Local Constraint Manipulation in Factor Graphs](https://doi.org/10.3390/e23070807) describes theoretical aspects of the underlying Bayesian inference method.
+- [Reactive Message Passing for Scalable Bayesian Inference](https://doi.org/10.48550/arXiv.2112.13251) describes implementation aspects of the Bayesian inference engine and performs benchmarks & accuracy comparison on various models.
```

Comment:
...performs benchmarks and accuracy comparison... (PLEASE NEVER USE SIGNS LIKE '&' RATHER THAN 'AND' IN TEXT).